### PR TITLE
Main cleanup of Muon v 6.3 release notes

### PR DESCRIPTION
**Pull request by sf1919**
**Description of work.**
The main refactoring the the Muon release notes ahead of release of 6.3. Includes:-

- Removing unused headings and developer warning text
- Reordering of notes where appropriate
- Rewording notes to improve consistency
- Adding links
- Spelling & Grammar check
- Adding an image for new error shading feature

Please note that the Model Fitting notes at the end are currently invisible to users. Apart from moving some notes into this section and rewording all to Model Fitting for consistency no other editing has taken place. This is because we are awaiting approval for release of this feature. These notes will either be revamped or moved to v.6.4 release notes before release.

**To test:**

-     remove build/docs/doctrees
-     build the documents again and check there are no warnings
-     Check that the links work
-     Check the information is still correct
-     Are the images visible and clearly illustrate the new feature/improvement?
-     Grammar/Spelling check

*There is no associated issue.*

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: docs/source/release/v6.3.0/muon.rst [hunk 1]
```diff
@@ -5,77 +5,67 @@ Muon Changes
...
-
 
 Muon Analysis and Frequency Domain Analysis
 -------------------------------------------
 
 New Features
 ############
 
-- Instead of plotting the confidence interval of a fit as an error bar, it is now represented by a shaded region.
+- **Instead of plotting the confidence interval of a fit as an error bar, it is now represented by a shaded region.**
```
Comment (AnthonyLim23) [inline docs/source/release/v6.3.0/muon.rst:29]: is this supposed to be in bold?

File: docs/source/release/v6.3.0/muon.rst [hunk 2]
```diff
@@ -5,77 +5,67 @@ Muon Changes
...
+
 
 Improvements
 ############
 
-- The `alpha` values on grouping tab are now to six decimal places.
-- The numerical values in the `run info` box on the home tab are now rounded to either 4 significant figures or a whole number, whichever is more precise.
+- Changes have been made to improve the speed of Muon Analysis and Frequency Domain Analysis.
+- The :ref:`Results Tab <muon_results_tab-ref>` will now dispay a warning (red text and a tooltip) if the results table already exists.
 - The results table now produces errors for log values (when they are available).
```
Comment (AnthonyLim23) [inline docs/source/release/v6.3.0/muon.rst:41]: is it deliberate that above has a link for the results tab and this does not?

File: docs/source/release/v6.3.0/muon.rst [hunk 3]
```diff
@@ -85,35 +75,32 @@ Improvements
...
 -----------------
 New Features
 ############
-* Added two Activation fitting functions to MuonModelling Fit Functions.
+* Added two Activation fitting functions to ``MuonModelling`` Fit Functions.
 
-  * `ActivationK <func-ActivationK>` can be used for data in Kelvins.
-  * `ActivationmeV <func-ActivationmeV>`can be used for data in meV.
+  * :ref:`ActivationK <func-ActivationK>` can be used for data in Kelvins.
+  * :ref:`ActivationmeV <func-ActivationmeV>`can be used for data in meV.
```
Comment (AnthonyLim23) [inline docs/source/release/v6.3.0/muon.rst:86]: This link does not work

File: docs/source/release/v6.3.0/muon.rst (hunk 1 above)
Comment (sf1919) [reply docs/source/release/v6.3.0/muon.rst:29]: Any note that accompanies an image is supposed to be in bold.

File: docs/source/release/v6.3.0/muon.rst [hunk 4]
```diff
@@ -85,35 +75,32 @@ Improvements
...
 
-* Added a :ref:`Critical peak of relaxation rate <func-CriticalPeakRelaxationRate>` for fitting to MuonModelling\Magnetism Fit Functions.
+* Added a :ref:`Critical peak of relaxation rate <func-CriticalPeakRelaxationRate>` for fitting to ``MuonModelling\Magnetism`` Fit Functions.
 * Added two fitting functions for the decoupling of asymmetry in the ordered state of a powedered magnet for fitting.
 
-  * `DecoupAsymPowderMagLong <func-DecoupAsymPowderMagLong>` can be used for longitudinal polarization.
-  * `DecoupAsymPowderMagRot <func-DecoupAsymPowderMagRot>`can be used for rotational asymmetry.
+  * :ref:`DecoupAsymPowderMagLong <func-DecoupAsymPowderMagLong>` can be used for longitudinal polarization.
+  * :ref:`DecoupAsymPowderMagRot <func-DecoupAsymPowderMagRot>`can be used for rotational asymmetry.
```
Comment (AnthonyLim23) [inline docs/source/release/v6.3.0/muon.rst:92]: This link does not work

File: docs/source/release/v6.3.0/muon.rst [hunk 5]
```diff
@@ -85,35 +75,32 @@ Improvements
...
 
-* Added a :ref:`Magentic Order Parameter<func-MagneticOrderParameter>` function to MuonModelling\Magentism Fit Functions.
-* Added a :ref:`Muonium-style Decoupling Curve <func-MuoniumDecouplingCurve>` function to MuonModelling Fit Functions.
-* Added a :ref:`Power Law <func-PowerLaw>` fitting function to MuonModelling Fit Functions.
-* Added a ref:`Smooth Transition <func-SmoothTransition>` function to MuonModelling Fit Functions.
+* Added a :ref:`Magentic Order Parameter<func-MagneticOrderParameter>` function to ``MuonModelling\Magentism`` Fit Functions.
```
Comment (AnthonyLim23) [inline docs/source/release/v6.3.0/muon.rst:94]: I followed the link and the equation is not formatted correctly

File: docs/source/release/v6.3.0/muon.rst (hunk 2 above)
Comment (sf1919) [reply docs/source/release/v6.3.0/muon.rst:41]: Are the results table and the result tab the same thing? I wasn't sure.

Comment (AnthonyLim23) [reply docs/source/release/v6.3.0/muon.rst:41]: ah no, sorry I misread it (missed the le)

File: docs/source/release/v6.3.0/muon.rst (hunk 3 above)
Comment (sf1919) [reply docs/source/release/v6.3.0/muon.rst:86]: Is it both links that don't work or just the second one?

File: docs/source/release/v6.3.0/muon.rst (hunk 4 above)
Comment (sf1919) [reply docs/source/release/v6.3.0/muon.rst:92]: Again is this both links or just the second one?

Comment (AnthonyLim23) [reply docs/source/release/v6.3.0/muon.rst:92]: the second in both cases

Comment (AnthonyLim23) [reply docs/source/release/v6.3.0/muon.rst:92]: I think its just a formatting problem

Comment (sf1919) [reply docs/source/release/v6.3.0/muon.rst:92]: Yeah its because there is no space after the ` at the end of the link and the next word.

**Comment by AnthonyLim23** [comment]
the decoupling functions have mangled equations in their docs

File: docs/source/release/v6.3.0/muon.rst (hunk 5 above)
Comment (AnthonyLim23) [reply docs/source/release/v6.3.0/muon.rst:94]: also A0 should be A_0

**Review by AnthonyLim23 — APPROVED** [review]
Looks good to me

**Pull request merged**